internal/pkg/template/templates/workloads/services/backend/manifest.yml

```diff
@@ -40,7 +40,11 @@ memory: {{.Memory}}    # Amount of memory in MiB used by the task.
 platform: {{.Platform}}     # See https://aws.github.io/copilot-cli/docs/manifest/backend-service/#platform
 {{- end}}
 count: {{.Count.Value}}       # Number of tasks that should be running in your service.
+  {{- if not .Platform}}
 exec: true     # Enable running commands in your container.
+  {{- else if ne .Platform.PlatformString "windows/amd64"}}
+exec: true     # Enable running commands in your container.
```

Comment:
Good question. No, because when we detect Windows, we autopopulate with "windows/amd64" as a string. The user then has the option to specify the "OS Family" (default will be Core) as a map along with Architecture. 

❤️  the "- if or"! TY.

Reply:
Update: have to revert the clever "if or"-- nil pointer error when trying to evaluate it if platform is nil.